src/Gui/DockWindowManager.cpp

```diff
@@ -380,6 +380,20 @@ void DockWindowManager::setup(DockWindowItems* items)
         }
     }
 
+    // Don't always tabify after switching the workbench
+    static bool tabify = false;
+    if (!tabify) {
+        // tabify dock widgets for which "tabbed" is true and which have the same position
+        for (int i=0; i<4; i++) {
+            const QList<QDockWidget*>& dws = areas[i];
+            for (QList<QDockWidget*>::ConstIterator it = dws.begin(); it != dws.end(); ++it) {

```

Comment:
Are you against for range-based for loop ? :smile:

Reply:
No. In new code I use range-based loops where ever possible. This is just the old code that has been removed a year ago and I have put it back.

Reply:
Right. `areas` is a local variable so we shouldn't expect much problems changing its type. Let you see what you think is best.

Reply:
Replaced the iterators with a range-based loop. But this will have happened anyway when continuing my work to modernize the C++ code base.